### DISCUSSION TITLE
Meteo pour home trainer

### DISCUSSION THREAD

**Discussion by Patlecoureur** [mytourbook/mytourbook#1167] [Ideas]
Bonjour, utilisateur de MyTourBook depuis plusieurs années, j'en suis très content et me permet de poser une question concernant la météo. L'application récupère le positionnement GPS pour trouver la météo du lieu et du moment, mais comme je fais pas mal de home trainer en période hivernale ou pluvieuse avec l'application Zwift, le parcours effectué se situe bien souvent en dehors de ma région et fatalement indique une météo erronée , je suis donc obligé de rentrer manuellement les différentes valeurs. L'idée serait de pouvoir changer le positionnement GPS afin d'obtenir la bonne météo. Est ce que cela existe déjà et que je ne l'aurait pas vu ? ou une autre méthode ? Merci d'avance et encore bravo a vous pour votre travail. Sportivement

**Comment by FJBDev** [comment]
Bonjour,

Ce nest pas une fonctionnalite disponible aujoudhui.
Le probleme etant : comment savoir quelles coordonneew GPS utiliser? Et de plus, la meteo exterieure est differente de la meteo interieure et donc non representative?

**Reply by Patlecoureur** [reply]
Bonjour et merci pour votre réponse. 
Ce nest pas une fonctionnalite disponible aujoudhui : oui, je m'en doutais un peu !! et c'est pour cela que j'ai posté dans la section "idée"
comment savoir quelles coordonnee GPS utiliser?: peut être en utilisant le fuseau horaire sélectionné.
Et de plus, la meteo exterieure est differente de la meteo interieure: C'est la météo extérieure qui est intéressante afin d'avoir ces valeurs pour les statistiques météo du coup je rentre manuellement les valeurs extérieure de mon lieu habitation ( pour savoir quel temps il faisait dehors le jour de la session zwift), donc peut etre utiliser le s valeurs météo en fonction du fuseau horaire (Paris ou Luxemnourg dans mon cas)
J'imagine que cela ne doit pas être simple à programmer et si je suis le seul à faire cette demande, alors oubliez !!
merci encore et bonne journée

**Reply by FJBDev** [reply]
Bonjour,

>> comment savoir quelles coordonnee GPS utiliser?: peut être en utilisant le fuseau horaire sélectionné.

Je ne suis pas sur que cela marcherait. Par exemple, pour les activites Zwift, le fuseau horaire du lieu virtuel serait utilise et donc different de celui du Luxembourg.
Si les activites importees ne contenaient pas de donnees GPS, alors le fuseau horaire par defaut pourrait etre utilise mais lorsque l'activite importee contient des donnees GPS, je ne vois pas comment MT peut detecter qu'un autre fuseau horaire doit etre utilise pour recuperer la meteo.

Frederic
